services/web-server/src/PulseEngine/Subscription.js

```diff
@@ -0,0 +1,162 @@
+import Debug from 'debug';
+
+const debug = Debug('Subscription');
+
+export default class Subscription {
+  constructor({ subscriptionId, handleMessage, handleError, subscriptions }) {
+    this.subscriptionId = subscriptionId;
```

Comment:
The constructor is missing `this.consumerTag`. Also, it'd be great if you could add a comment explaining `consumerTag`.